pina/model/layers/avno.py

```diff
@@ -0,0 +1,28 @@
+"""Module for Averaging Neural Operator Layer class."""
+
+from torch import nn, mean
+
+class AVNOBlock(nn.Module):
+    """
+    The PINA implementation of the inner layer 
+        of the Averaging Neural Operator . 
+
+    :param int hidden_size: size of the layer.
+        Defaults to 100.
+    :param func: the activation function to use. 
+        Default to nn.GELU.
+        
+    """
+    def __init__(self, hidden_size = 100, func = nn.GELU):
```

Comment:
my suggestion is to instantiate the class of `func` in the init and not at every forward pass, but I do not think it worsen the performance